Cilsil/Cil/Parsers/ExceptionParser.cs

```diff
@@ -149,6 +149,7 @@ internal class ExceptionParser : InstructionParser
             // Construct a finally block when unwrapped exception type is "System.Object".
             else
             {
+                state.PushRetExpr();
```

Comment:
We simplified loading returned variable into exception handling blocks (`catch`, `finally`) by pushing a [dummy expression] (https://github.com/xi-liu-ds/infersharp/blob/6abf2af55d3c2d4a6d24bb3e1233805af94a303b/Cilsil/Utils/ProgramState.cs#L169)